### PR TITLE
test(client): use `Providers` instead of strings

### PR DESCRIPTION
**Pull request by Jolg42**
It's not an exhaustive cleanup, but a good start.

### CODE REVIEW

File: packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts [hunk 1]
```diff
@@ -24,7 +25,7 @@ testMatrix.setupTestSuite(
         /client/tests/functional/0-legacy-ports/malformed-id/tests.ts:0:0
 
            XX testMatrix.setupTestSuite(
-           XX   () => {
+          XX   () => {
```
Comment (Druue) [inline packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts:28]: random removed white-space 🤔 probably not related to the PR itself?

**Review by Druue — APPROVED** [review]

File: packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts (hunk 1 above)
Comment (Jolg42) [reply packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts:28]: It is related and needed here actually.

Comment (Jolg42) [reply packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts:28]: But it's really annoying ...

**Comment by SevInf** [comment]
Could you also adjust [template](https://github.com/prisma/prisma/blob/b333dcd4979d65dc34036d07f998566fb6f520c9/packages/client/helpers/new-test/new-test.ts#L62) for new tests?

**Comment by Jolg42** [comment]
Good idea @SevInf - I think you are the only one using it? 😄 😉 

I think we could merge this without this change though.

**Pull request merged**